pixart.js

```diff
@@ -1 +1,93 @@
+"use strict"
+$(document).ready(function() {
 
+  // Append Canvas to Body
+  var appendCanvas = function() {
+    // Create Canvas
+    for (var i = 0; i < 8000; i++) {
+      $("body").append("<div class='square'></div>");
+    }
+    // Create Test Div
+    $(".controls").append("<div id='test'></div>");
+  }
+
+  // Create Color Swatch Palette
+  var createSwatch = function() {
+    for (var i = 0; i < 3; i++) {
+      $(".controls").append("<div class='swatch'></div>");
+    }
+    $(".swatch").css({
+      "display": "inline-block",
+      "height": "25px",
+      "width": "25px",
+      "margin": "5px",
+    });
+    swatchColorSet();
+  }
+
+  var swatchColorSet = function() {
+    $(".swatch").each(function(index){
+      $(this).css("background-color", allLiveColors[index]);
+    })
+  }
+
+  // Set Brush Color Variables
+  var setColorButton = $("#set-color");
+  var brushColorBox = $(".brush");
+  var brushColor = "#1B4370";
+  var allLiveColors = ["darkcyan", "rebeccapurple", "palevioletred", "#1B4370"];
+
+  // User Input Set Color
+  var setColor = function(evt) {
+    evt.preventDefault();
+    if (testColor()) {
+      brushColor = $("#color-field").val();
+      rotatePalette();
+    }
+  }
+
+  // User Selects Color From Color Swatch Palette
+  var setColorFromPalette = function() {
+    brushColor = $(this).css("background-color");
+    rotatePalette();
+  }
+
+  // Rotate Color Swatch Palette
+  var rotatePalette = function() {
+    allLiveColors.shift()
+    allLiveColors.push(brushColor);
+    swatchColorSet();
+    brushColorBox.css("background-color", brushColor);
+  }
+
+  //Test Color Function (Validates User Input)
+  var testColor = function() {
+      $("#test").css("background-color", brushColor);
+      var testThis = $("#color-field").val();
+      $("#test").css("background-color", testThis);
+      if ($("#test").css("background-color") != brushColorBox.css("background-color")) {
+        return true;
+      } else {
+        return false;
+      }
```

Comment:
Can get rid of the if-else statement by just returning the comparison evaluated on line 68:
`return ($("#test").css("background-color") != brushColorBox.css("background-color"))`